redaxo/src/addons/structure/plugins/content/assets/content.js

```diff
@@ -1,5 +1,5 @@
 // save & restore sidebar panel toggle status via localstorage
-jQuery(function($){
+$(document).on('rex:ready', function (event, viewRoot) {
     var $sidebar = $('#rex-page-content-edit #rex-js-main-sidebar'),
```

Comment:
Sobald man mit `rex:ready` arbeitet, ist es in der Regel verkehrt, innerhalb dann `$()` zu verwenden.
Sondern in der regel dann `viewRoot.find()`, also nur innerhalb des aktualisierten Containers.
Ich glaube, das würde hier auch passen.

Reply:
habe es angepasst.